### PR TITLE
Add executive summary

### PR DESCRIPTION
**Pull request by sdruskat**
As discussed in today's meeting, this PR adds an exec summary to the survey.
It also fixes some formatting issues which appeared when rendered in mdbook.
Please feel free to merge this if you're happy with it.

After this is merged, your PR can then be merged to the main hexatomic repo.

### CODE REVIEW

**Comment by sdruskat** [comment]
Good to go from my side. @thomaskrause, have a look at the conclusion as well, please, to make sure I haven't forgotten anything important.

File: src/architecture/extensibility/nlp-survey.md [hunk 1]
```diff
@@ -22,9 +22,27 @@ Specifically, the following properties were checked:
...
 
-### [AllenNLP](<https://allennlp.org/>)
+This left us with four potential candidates: [Apache OpenNLP](#apache-opennlp), [SpaCy](#spacy), [Spark NLP](#spark-nlp), and [Stanford CoreNLP](#stanford-corenlp).
+
+Of those, the only candidate in Java that is feature-complete was Stanford CoreNLP:
+Apache OpenNLP did not seem to support dependency parsing, SparkNLP as well as SpaCy did not seem to support constituency parsing (directly).
+
+In the end, we settled on integration of Stanford CoreNLP in Hexatomic.
+It seems to cater best for the core target group of early versions of Hexatomic, i.e., users with a strong linguistic rather than NLP background.
+It also seemed architecturally easiest to integrate, as we will not have to bridge between Java dn Python code at this stage.
```
Comment (thomaskrause) [inline src/architecture/extensibility/nlp-survey.md]: Typo
```suggestion
It also seemed architecturally easiest to integrate, as we will not have to bridge between Java and Python code at this stage.
```

**Review by thomaskrause — APPROVED** [review]
Looks good to me, found a single typo

**Comment by sdruskat** [comment]
@clachenmaier Good to go then, feel free to merge, so that we can merge back to the main repo. Thanks!

**Pull request merged**